docs/src/installing-deis/installing-the-deis-platform.md

```diff
@@ -35,78 +35,156 @@ $ kubectl get pods --namespace=deis
+choosing, and save changes.  Kubernetes will apply the changes and
+the router will dynamically reconfigure itself accordingly.
+
+### Getting Traffic to the Cluster
+
+With the platform domain configured, the only remaining concern
+is to ensure traffic for that domain reaches the cluster.  There
+are a few ways of doing this, and which you use may depend on the
+capabilities of your infrastructure and your Kubernetes cluster.
+
+In general, the goal is to connect traffic bound for
+`*.the-domain-you.picked` to your Kubernetes cluster.
+
+#### Using an Automatically Provisioned Load Balancer
+
+If your Kubernetes cluster and the underlying infrastructure
+support it, the Deis router's service (which is of `type:
+LoadBalancer`) will automatically provision an "external"
+(external to the Kubernetes cluster) load balancer.  On AWS,
+for instance, this takes the form of an ELB, while on Google
+Container Engine, it takes the form of a routing rule.
+
+If it is supported, you will be able to find the DNS name or
+IP(s) of the external load balancer like so:
+
+```
+$ kubectl describe service deis-router --namespace=deis
+```
+
+Examine the `LoadBalancer Ingress` field to find the DNS
+name or IP(s).  The former can be added to DNS as a CNAME,
+while the latter can be added as an A record.
+
+#### Creating Your Own Load Balancer
+
+On Kubernetes clusters and underlying infrastructure that
+do not support the automated provisioning of an external
+load balancer for the Deis router, you may provision your
+own and add _all_ worker nodes of the Kubernetes cluster to
+the pool.
+
+The load balancer must pass incoming traffic on the
+following ports to the _same_ ports on the Kubernetes
+worker nodes:
+
+* 80
+* 443
+* 2222
+* 9090
+
+The Deis router component listens to those ports on any
+host it runs on.  Since not _all_ nodes are likely to
+be running the router, a health check should be used to
+determine which node(s) can serve requests.  The router's
+healthcheck should use the HTTP protocol, port `9090`, and
+the request path `/healthz`.
+
+Load balancer idle timeouts should be set to at least `1200`
+seconds so connections do not close prematurely during longer
+running requests such as application deployments.
+
+#### Without a Load Balancer
+
+On some platforms (Vagrant, for instance) a load balancer is
+not an easy or practical thing to provision.  In these cases,
+one can directly identify the public IP of a Kubernetes node
+that is hosting a router pod and use that information to
+configure DNS or the local `/etc/hosts` file.
 
 You can find the IP address of a node using `kubectl`:
 
 ```
-$ kubectl get nodes
-NAME         LABELS                              STATUS    AGE
-10.245.1.3   kubernetes.io/hostname=10.245.1.3   Ready     19h
+$ kubectl get pods --namespace=deis | grep deis-router
+NAME                        READY     STATUS    RESTARTS   AGE
+deis-router-ih25q           1/1       Running   0          2h
```

Comment:
I actually knew that... I deliberately put it there because I thought it added valuable context and didn't think anyone would notice.  You win a prize @sgoings!